odoo/addons/base/views/ir_actions_views.xml

```diff
@@ -342,6 +342,7 @@
                         <div class="d-flex flex-row flex-wrap gap-2" invisible="state != 'object_write'">
                             <field name="evaluation_type" class="oe_inline"/>
                             <field name="update_path" widget="DynamicModelFieldSelectorChar" class="oe_inline" options="{'model': 'model_name'}"/>
+                            <field name="update_related_model_id" invisible="True"/>  <!-- It is required by the ReferenceField -->
```

Comment:
It's already fixed: https://github.com/odoo/odoo/blob/saas-17.4/odoo/addons/base/views/ir_actions_views.xml#L346